### DISCUSSION TITLE
Postgresql - Clickhouse - CDC problem

### DISCUSSION THREAD

**Discussion by grig0701** [airbytehq/airbyte#36463] [Connector Questions]
### Topic

Postgresql - Clickhouse - CDC

### Relevant information

We have source postgres and destination clickhouse, also update method was selected as CDC. Please tell me why only two sync methods are available for tables with such a configuration, should it be so?
<img width="224" alt="image" src="https://github.com/airbytehq/airbyte/assets/125663577/20ba798e-5aa7-445e-b5f9-3f1c83d202d4">

**Comment by Lawchak-Fadhil** [comment]
Hello, I don't really have an answer for your question but I see that you have a similar setup to mine, PostgreSQL > CH using Airbyte. I was wondering if you are getting desired results when you are syncing tables this way? I mean if you have a table "test" on the PostgreSQL, when you sync to your destination CH, do you get the same table with same name and structure? I am trying to achieve this but I can't get it to do so for some reason..

[a comment by an automated account is not shown]
